### ISSUE TITLE
Put ">" after the last attribute when wrapping multiple attributes

### ISSUE THREAD

**Issue by lhajh** [vuejs/vetur#1000]
- [✅ ] I have searched through existing issues
- [✅ ] I have read through [docs](https://vuejs.github.io/vetur)
- [ ✅] I have read [FAQ](https://github.com/vuejs/vetur/blob/master/docs/FAQ.md)

## Info

- Platform: macOS
- Vetur version: 0.14.2
- VS Code version: 1.29.1

## Problem

For multiple attributes that extend beyond the printWidth, I would prefer this:

```html
<p
  class="class"
  title="title">
  <span
    class="class"
    title="title"
  >test</span>
</p>
```

not this:

```html
<p
  class="class"
  title="title"
>
  <span
    class="class"
    title="title"
  >test</span>
</p>
```

Although this is prettier than the one above, but fold code is a hassle, so can you add a configuration to choose between this two formatting options?

**Comment by octref** [comment]
You should open an issue for https://github.com/Prettyhtml/prettyhtml, although I don't think @StarpTech would change this. There are two options and as an opinionated formatter, Prettyhtml has to choose one. He chose this format and I like it.

The folding issue is valid. Vetur currently only has indentation based folding, but not intelligent one. I can add it and it would fix this issue.

**Comment by cepm-nate** [comment]
Any update on intelligent folding for vue/js/html/css files with vetur loaded? Currently my code goes from this
```
actions: {
    /* some stuff /*
}
```
to this (FoldingStrategy set to auto)
```
actions: { ...
}
```
But I'd really like to see it all on one line, like
```
actions: { ... }
```

Oddly enough, `[]` collapse into one line while `{}` do not.
Thanks!

**Comment by cepm-nate** [comment]
According to https://github.com/Microsoft/vscode/issues/3352, this feature will need to be implemented as an extension. 

They point to https://stackoverflow.com/questions/36825024/can-you-customize-code-folding (showing an example), which uses https://code.visualstudio.com/api/references/vscode-api#FoldingRangeProvider.

**Comment by octref** [comment]
Yeah, but that logic lives in the TypeScript Language Server, which doesn't support this yet:

![image](https://user-images.githubusercontent.com/4033249/54788687-2e70fd80-4bed-11e9-8e49-d4b0f6968704.png)

I wouldn't have time to do this myself.
